chorus_book/src/guide-input-and-output.md

```diff
@@ -42,7 +42,8 @@ You can construct an instance of the choreography with the input and pass it to
 let choreo = DemoChoreography {
     input: "World".to_string(),
 };
-let projector = Projector::new(Alice, transport);
+type AL = hlist!(Alice);
```

Comment:
I feel like "location set" is a better name and personally want to stop using "available locations" but I also want your opinion.

```rust
projector!(locations!(Alice, Bob), Alice, transport)
```

Reply:
I'm used to `available locations` at this point but I can see that `location set` can be more understandable for people who just start using it. So yeah, I think we should change that.

But I'm assuming the actual implementation will still be called `HList`?